Cargo.toml

```diff
@@ -7,12 +7,36 @@ edition = "2021"
 
 [dependencies]
 bytesize = "1.1"
+clap = { version = "3", features = ["derive"] }
+dirs = "4"
 either = "1.8"
 futures = "0.3"
-libp2p-core = "0.36"
+libp2p-core = "0.34"
```

Comment:
curious, why the version bump down?

Reply:
Cause of the version conflict. Right now monorepo uses `libp2p-core = 0.34`